Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Certificate_Enrichment.yml

```diff
@@ -0,0 +1,782 @@
+description: Playbook to enrich certificate information.
+id: 'Cortex ASM - Certificate Enrichment'
+inputSections:
+- description: Generic group for inputs
+  inputs:
+  - Hostname
+  name: General (Inputs group)
+inputs:
+- description: Input for Certificate enrichment
+  key: Hostname
+  playbookInputQuery:
+  required: false
+  value:
+    simple: ${alert.hostname}
+name: Cortex ASM - Certificate Enrichment
+outputSections:
+- description: Generic group for outputs
+  name: General (Outputs group)
+  outputs: []
+outputs: []
+starttaskid: "0"
+tasks:
+  "0":
+    continueonerrortype: ""
+    id: "0"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "16"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 0804bf72-274c-489c-86fa-0c78f7930d42
+      iscommand: false
+      name: ""
+      version: -1
+      description: ''
+    taskid: 0804bf72-274c-489c-86fa-0c78f7930d42
+    timertriggers: []
+    type: start
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -80
+        }
+      }
+  "1":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              complex:
+                filters:
+                - - left:
+                      iscontext: true
+                      value:
+                        simple: modules.brand
+                    operator: isEqualString
+                    right:
+                      value:
+                        simple: VenafiTLSProtect
+                - - left:
+                      iscontext: true
+                      value:
+                        simple: modules.state
+                    operator: isEqualString
+                    right:
+                      value:
+                        simple: active
+                root: modules
+          operator: isExists
```

Comment:
Usually, I will suggest using `Is Not Empty` - but if that works for you, then we should be fine.
The main reason for using the Is not Empty is to avoid situations when the key will be created but without a value.

Reply:
Thank you for the suggestion. We usually select "Is defined" as an operator in the playbook and it shows up as "isExists" in the yml file. 

But, I will bring this up with the team and probably start using this suggestions going forward.